MontyTRC89/TombEngine
# Bridge refactor

Adds `BridgeObject` `ItemData` variant and moves bridge routines from `ObjectInfo` into it.

For code review:
- Check if all bridge objects have been accounted for in the object ID collection.
- Maybe there's a better way to do this? To load bridges before everything else, I substitute a hack for a hack.

To test:
- Climbable pushables placed on bridges on level load.
- Non-bridge moveables placed on bridges on level load.
- Ensure bridges still work the same as before.
